lookup.go

```diff
@@ -20,8 +20,11 @@ func (dht *IpfsDHT) GetClosestPeers(ctx context.Context, key string) ([]peer.ID,
 	if key == "" {
 		return nil, fmt.Errorf("can't lookup empty key")
 	}
-	// TODO: I can break the interface! return []peer.ID
-	lookupRes, err := dht.runLookupWithFollowup(ctx, key,
+
+	const isHashed = false
```

Comment:
will we add a check to see if it's hashed later on?

Reply:
yeah, it's checked in `query.go` line 152

Reply:
actually, this got removed since the double hash is now a `multihash.Multihash`